testsuite/tests/language-extensions/language_extensions.ml

```diff
@@ -0,0 +1,163 @@
+(* TEST
+   include ocamlcommon
+   flags = "-I ${ocamlsrcdir}/parsing"
+   reference = "${test_source_directory}/reference.txt"
+   * skip
```

Comment:
What's going on here? This is a big new test, but it's skipped. I did not read it.

Reply:
The reason is on the line below!  I completely forgot about this.  There should be a comment somewhere but I'm not sure where it went.  This is the danger in making commits marked "Temporary".  What's going on here: I wrote this test back when `local_` only parsed in the presence of `-extension local`.  Now, it always parses and fails later.  There are two changes we can make: (1) make `local_` only parse in the presence of `-extension local` again, or (2) use a modular extension language extension.  I think I'll make the change to (2), but then it's really only testing the modular extension machinery; I think that's probably fine.

Reply:
This should be reviewable now